build-tools/automation/azure-pipelines.yaml

```diff
@@ -165,6 +167,14 @@ stages:
     - template: productsign-pkg.yml@yaml
       parameters:
         UnsignedPkgPath: $(XA.Unsigned.Pkg)
+    
+    - script: |
+        cd $(Build.SourcesDirectory)/..
+        git clone -b $(ReleaseScriptsBranch) https://$(GitHub.Token):x-oauth-basic@github.com/xamarin/release-scripts
+        cd release-scripts
+        ruby notarize.rb $(XA.Unsigned.Pkg) $(XamarinIdentifier) $(XamarinUserId) $(XamarinPassword) $(TeamID)
+      displayName: Notarize PKG
+      condition: and(succeeded(), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/master' ), startsWith(variables['Build.SourceBranch'], 'refs/heads/release' )))
```

Comment:
Let's update this to use the same "Is not pull request" condition used elsewhere, as our release branches historically don't start with `release`:
```yaml
condition: and(eq(variables['XA.Commercial.Build'], 'true'), ne(variables['Build.Reason'], 'PullRequest'))
```

Reply:
Yes the script handles stapling. I don't think we should be creating a template for just one task.

Reply:
Pushed the change for the condition

Reply:
Fair enough 👍